docs/oss/issues.md

```diff
@@ -25,8 +25,14 @@ Here all requests for Eclipse infrastructure go in.
 
 **IMPORTANT:** only a committer can create a valid IP Ticket!
 
-- IP issue for Code Contributions: [Template](https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/new?issuable_template=vet-project), choose template "vet-project" ([more information](/docs/release/trg-7/trg-7-03))
-- IP issue for 3rd party content: [Template](https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/new?issuable_template=vet-third-party), choose template "vet-third party" ([more information](/docs/release/trg-7/trg-7-04#checking-other-content-fonts-images-))
+- IP issue for [Code Contributions](https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/new), choose template "vet-project"
+  - [more information](/docs/release/trg-7/trg-7-03)
+  - title pattern: `project/<project id>/-/<name>/0.0`,
+  - [Example](https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/8097)
+  - For PRs: use the link to the PR as source reference!
+- IP issue for [3rd party content](https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/new), choose template "vet-third party":
+  - [more information](/docs/release/trg-7/trg-7-04#checking-other-content-fonts-images-)
+  - [Example](https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/5875)
```

Comment:
should we take here an example form an actual IP Issue with correct description 'tractus-x' as prefix for the IP Issue?

Reply:
IP issues have defined titles. The project name / id has to be sumitted in the form.